### PR TITLE
Add Apache Pulsar scaler docs.

### PR DESCRIPTION
**Pull request by liangyuanpeng**
Co-authored-by: mmussett mmussett@me.com
Signed-off-by: Lan Liang <gcslyp@gmail.com>

_Provide a description of what has been changed_  

Base on https://github.com/kedacore/keda-docs/pull/397/files , So i add a `co-authored-by` for @mmussett 

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.8/scalers/pulsar.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    subscription: sub1
+    msgBacklogThreshold: '5'
+```
+
+**Parameter list:**
+
+- `adminURL` - Stats URL of the admin API for your topic.
+- `topic` - Pulsar topic. format of `persistent://{tenant}/{namespace}/{topicName}`
+- `subscription` - Name of the topic subscription
+- `msgBacklogThreshold` - Average target value to trigger scaling actions. (default: 10)
```
Comment (liangyuanpeng) [inline content/docs/2.8/scalers/pulsar.md:28]: Reply for https://github.com/kedacore/keda-docs/pull/397/files#r593432141  , `msgBacklog`  is the full name of message backlog in the pulsar, so i have not change it.

Comment (tomkerkhove) [reply content/docs/2.8/scalers/pulsar.md:28]: Definitely understand but we tend not to use abbreviations so prefer to use messageBacklogTreshhold instead

**Review by JorTurFer — COMMENTED** [review]
Could you add the activation parameter too?

File: content/docs/2.8/scalers/pulsar.md (hunk 1 above)
Comment (zroubalik) [reply content/docs/2.8/scalers/pulsar.md:28]: +1 for keeping the abbreviation because to follow upstream

Comment (zroubalik) [inline content/docs/2.8/scalers/pulsar.md:28]: `activationMsgBacklogThreshold` is missing here, please see other scaler's on the proper wording of the description, for example: https://github.com/kedacore/keda-docs/blob/main/content/docs/2.8/scalers/activemq.md

File: content/docs/2.8/scalers/pulsar.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+This specification describes the `pulsar` trigger for an Apache Pulsar topic.
+
+```yaml
+triggers:
+- type: pulsar
+  metadata:
+    adminURL: http://localhost:80
+    topic: persistent://public/default/my-topic
+    subscription: sub1
+    msgBacklogThreshold: '5'
```
Comment (zroubalik) [inline content/docs/2.8/scalers/pulsar.md:20]: `activationMsgBacklogThreshold` is missing here.

**Review by zroubalik — CHANGES_REQUESTED** [review]

File: content/docs/2.8/scalers/pulsar.md (hunk 2 above)
Comment (zroubalik) [inline content/docs/2.8/scalers/pulsar.md:20]:
```suggestion
    msgBacklogThreshold: '5'
    activationMsgBacklogThreshold: '2'
```

File: content/docs/2.8/scalers/pulsar.md (hunk 1 above)
Comment (zroubalik) [inline content/docs/2.8/scalers/pulsar.md:28]:
```suggestion
- `msgBacklogThreshold` - Average target value to trigger scaling actions. (default: 10)
- `activationMsgBacklogThreshold` - Target value for activating the scaler. Learn more about activation [here](./../concepts/scaling-deployments.md#activating-and-scaling-thresholds).(Default: `0`, Optional)
```

**Pull request merged**